relay-metrics/src/bucket.rs

```diff
@@ -735,6 +744,43 @@ impl CardinalityItem for Bucket {
     }
 }
 
+/// Relay internal metadata for a metric bucket.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct BucketMetadata {
+    /// How many times the bucket was merged.
+    ///
+    /// Merging two buckets sums the amount of merges.
+    pub merges: NonZeroU64,
```

Comment:
nit: I don't have a better name, but it would be nice to clarify somehow that this is the number of merges + 1 for the creation of the bucket.